Update schedule to reuse HDD also on patterns

ilmanzo opened the pull request:
- Related ticket: https://jira.suse.com/browse/TEAM-10236
- Needles: no
- Verification runs:

Minimal-VM-for-kvm-and-xen-SAP:
  - https://openqa.suse.de/tests/17674158
  - https://openqa.suse.de/tests/17674159
  - https://openqa.suse.de/tests/17674160
  - https://openqa.suse.de/tests/17674161
  - https://openqa.suse.de/tests/17674162
  - https://openqa.suse.de/tests/17674163
  - https://openqa.suse.de/tests/17674164

Minimal-VM-Cloud-SAP:
  - https://openqa.suse.de/tests/17674273
  - https://openqa.suse.de/tests/17674274
  - https://openqa.suse.de/tests/17674275
  - https://openqa.suse.de/tests/17674276
  - https://openqa.suse.de/tests/17674277
  - https://openqa.suse.de/tests/17674287

as stated in the ticket AC, test failures will be addressed in a later step

mpagot commented on schedule/sles4sap/minimalvm/create_hdd.yaml:
```diff
@@ -1,41 +1,40 @@
-    - '{{wizard}}'
-    - console/journal_check
-    - console/suseconnect_scc
-    - sles4sap/patterns
```
so "`pattern` has been removed" is the main difference. Right?

ilmanzo replied:
correct, see discussion on https://suse.slack.com/archives/C0369JZFBKK/p1746795917152719

mpagot reviewed (approved):
LGTM

a-kpappas commented:
Why did you change the quotes? It pollutes the diff.

a-kpappas commented:
> > Why did you change the quotes? It pollutes the diff.
> 
> whops, good catch! Misconfigured editor :)

It also changed the whitespace 😅

ilmanzo commented:
> > > Why did you change the quotes? It pollutes the diff.
> > 
> > 
> > whops, good catch! Misconfigured editor :)
> 
> It also changed the whitespace 😅

fixed :D 

reminds me of https://youtu.be/oRva7UxGQDw

alvarocarvajald reviewed (approved):
LGTM
